packages/synapse-react-client/src/utils/functions/SqlFunctions.ts

```diff
@@ -148,18 +148,17 @@ export const parseEntityIdAndVersionFromSqlStatement = (
   }
 }
 
-export const resultToJson = <T>(
```

Comment:
This was improper use of a generic. It's not guaranteed that the result will match whatever you need it to.